src/app/components/views/weather-view/weather-view.component.scss

```diff
@@ -0,0 +1,18 @@
+// .rain
```

Comment:
this is all commented, it can be removed

Reply:
uncommented, it works but i was testing a few things.